src/tests/fuzzing/Makefile.am

```
@@ -18,6 +19,11 @@ if ENABLE_OPENSSL
 noinst_PROGRAMS += fuzz_piv_tool
 endif
 
+if ENABLE_SHARED
+else
+noinst_PROGRAMS += fuzz_pkcs11
+endif
```

would negative condition like this work?
```suggestion
if ! ENABLE_SHARED
noinst_PROGRAMS += fuzz_pkcs11
endif
```
Is there something preventing the build when the shared build is enabled? Should not this be in `if ENABLE_STATIC` above?

---

Yes, it can be simplified, but for the second comment, the fuzz target is linked with `libopensc-pkcs11.la`, which is built only when the shared build is not enabled (this part is similar to [`pkcs11-tool`](https://github.com/OpenSC/OpenSC/blob/master/src/tools/Makefile.am#L64-L68)).

---

Thanks fr clarification.